web/src/popover_menus.js

```diff
@@ -36,18 +40,45 @@ import {user_settings} from "./user_settings";
 let left_sidebar_stream_setting_popover_displayed = false;
 let compose_mobile_button_popover_displayed = false;
 export let compose_enter_sends_popover_displayed = false;
-let compose_control_buttons_popover_instance;
 let message_actions_popover_displayed = false;
 let message_actions_popover_keyboard_toggle = false;
 
+let compose_control_buttons_popover_instance;
+let starred_messages_popover_instance;
+
 export function actions_popped() {
     return message_actions_popover_displayed;
 }
 
+export function is_starred_messages_shown() {
+    return starred_messages_popover_instance?.state.isShown;
+}
+
 export function get_compose_control_buttons_popover() {
     return compose_control_buttons_popover_instance;
 }
 
+export function get_starred_messages_popover() {
+    return starred_messages_popover_instance;
+}
+
+function get_popover_items_for_class(class_name) {
+    const $current_elem = $("[data-tippy-root] " + class_name);
```

Comment:
Would this be cleaner to write basing off the `instance` for the popover rather than doing a lookup for a class from scratch?

Reply:
Yes, that would certainly be cleaner, thanks! Fixed!